### PR TITLE
Rebuild GraphQL section as API Usage

### PR DESCRIPTION
**Pull request by patrys**
New pages for pagination, authentication, internationalization, and usage limits.

### CODE REVIEW

**Comment by patrys** [comment]
Forgot to include a file 🤦

File: docs/api-usage/authentication.mdx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+## Authentication vs. authorization
+
+Authentication is the process of verifying the identity of an actor. Authorization is the process of verifying what permissions the actor has to access a resource.
+
+Saleor recognizes two types of actors:
+
+- **Users**: either customers or staff members (humans)
+- **Apps**: third-party integrations (programs)
+
+User tokens are short-lived and are used to access resources that are related to the user, such as their cart or orders.
```
Comment (CossackDex) [inline docs/api-usage/authentication.mdx:17]:
```suggestion
User tokens are short-lived and are used to access resources related to the user, such as their cart or orders.
```

File: docs/api-usage/authentication.mdx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+App tokens are long-lived and are used to access resources that are related to the app, such as products or orders.
```
Comment (CossackDex) [inline docs/api-usage/authentication.mdx:19]:
```suggestion
App tokens are long-lived and are used to access resources related to the app, such as products or orders.
```

File: sidebars.js [hunk 3]
```diff
@@ -46,17 +46,17 @@ module.exports = {
...
     {
       type: "category",
-      label: "GraphQL",
+      label: "API Usage",
       link: {
-        type: "doc",
-        id: "developer/getting-started/graphql",
+        type: "generated-index",
+        title: "API Usage",
+        description: "Saleor API conventions.",
```
Comment (lkostrowski) [inline sidebars.js:53]: I think this description text could be better, conventions?

File: docs/api-usage/i18n.mdx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+Fields that haven't been translated will return `null`.
+
+## Localizing prices and dates
+
+Prices and dates can be localized using [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) and [date-fns](https://date-fns.org).
+
+## Localizing address input
+
+Saleor provides an [`AddressInput`](api-reference/inputs/address-input.mdx) type that can be used to collect address data from the user. It contains fields for the country, city, postal code, and street address. Which fields should be displayed to the user and how to label them can be determined by querying the address validation API.
```
Comment (CossackDex) [inline docs/api-usage/i18n.mdx:78]:
```suggestion
Saleor provides an [`AddressInput`](api-reference/inputs/address-input.mdx) type that can be used to collect address data from the user. It contains country, city, postal code, and street address fields. Which fields should be displayed to the user and how to label them can be determined by querying the address validation API.
```

File: sidebars.js (hunk 3 above)
Comment (patrys) [reply sidebars.js:53]: Go ahead and improve it.

File: docs/api-usage/i18n.mdx (hunk 4 above)
Comment (patrys) [reply docs/api-usage/i18n.mdx:78]: I think it's easier to read this the way it was before. In your version, I have to get through the entire litany of names before I get to "fields" which is the whole point of this sentence.

File: docs/api-usage/overview.mdx [hunk 5]
```diff
@@ -1,17 +1,22 @@
...
 
+## Accessing the API
+
 The API endpoint is available at `/graphql/` and requires queries to be submitted using the HTTP `POST` method and the `application/json` content type.
 
 Using the Saleor GraphQL API allows you to query and modify all of your shop’s data flexibly and efficiently. Our API provides both types of operations:
 
-- Public (related to the customers of your shop)
-- Private (related to the administration side)
+- **Public**: carried out on behlf of customers of your storefront
```
Comment (CossackDex) [inline docs/api-usage/overview.mdx:18]:
```suggestion
- **Public**: carried out on behalf of customers of your storefront
```

**Review by lkostrowski — APPROVED** [review]

**Review by CossackDex — APPROVED** [review]

**Pull request merged**